BASE/src/Microsoft.ApplicationInsights/ActivityExtensions.cs

```diff
@@ -52,7 +52,11 @@ private static bool Initialize()
         {
             try
             {
+#if REDFIELD
```

Comment:
Can you open an issue to completely remove this class. it should not be required.

Reply:
Investigated a little and found methods from this class is called in several places.
For example Tryrun is called in `OperationCorrelationTelemetryInitializer.cs`. 

Did I understand your comment incorrectly?

Reply:
yea all the places where this ActivityExtensions is used can be modified to directly use Activity API, instead of doing it via this class.